modules/test/playwright/pages/commerce/commerce-product-content-web/productDetailsPage.ts

```diff
@@ -207,10 +208,14 @@ export class ProductDetailsPage {
 			this.ellipsisFrameProductSpecification.getByText(
 				'Success:Your request completed successfully.'
 			);
+		this.waitForAddWidgetSuccessMessage = page.getByText(
+			'Success:The application was added to the page.'
+		);
```

Comment:
can we use the waitForAlert instead of creating our ?

Reply:
sure, I forgot that we already have a generic solution to this.